internal/installation/paths.go

```diff
@@ -35,14 +35,15 @@ func InstallPath() (string, error) {
 
 	// If State Tool is already exists then we should detect the install path from there
 	stateInfo := appinfo.StateApp()
-	activeStateOwnedPath := strings.Contains(strings.ToLower(stateInfo.Exec()), "activestate")
-	if fileutils.TargetExists(stateInfo.Exec()) {
-		if filepath.Base(filepath.Dir(stateInfo.Exec())) == BinDirName && activeStateOwnedPath {
-			return filepath.Dir(filepath.Dir(stateInfo.Exec())), nil // <return this>/bin/state.exe
-		}
+	if !fileutils.TargetExists(stateInfo.Exec()) {
 		return filepath.Dir(stateInfo.Exec()), nil // <return this>/state.exe
 	}
 
+	activeStateOwnedPath := strings.Contains(strings.ToLower(stateInfo.Exec()), "activestate")
+	if filepath.Base(filepath.Dir(stateInfo.Exec())) == BinDirName && activeStateOwnedPath {
+		return filepath.Dir(filepath.Dir(filepath.Dir(stateInfo.Exec()))), nil // <return this>/<branch>/bin/state.exe
+	}
```

Comment:
What is this change fixing?

Reply:
This was a leftover from my initial attempts. Reverted now

Reply:
It doesn't look like it's reverted?

Reply:
Should be now